packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -711,31 +686,24 @@ function* swapStep({
   const beforeSwapAudioAccountInfo = yield* call(getAudioAccountInfo, {
     tokenAccount
   })
-  const beforeSwapAudioBalance =
-    beforeSwapAudioAccountInfo?.amount ?? new u64(0)
+  const beforeSwapAudioBalance = beforeSwapAudioAccountInfo?.amount ?? BigInt(0)
```

Comment:
nit: `0n` instead of `BigInt(0)` here imo

Reply:
getting this type error when using `0n`: https://stackoverflow.com/questions/57560501/bigint-literals-are-not-available-when-targeting-lower-than-esnext
Looks like we'd need to update our tsconfig.

Reply:
Oof, that's annoying. 😢